test-util/src/main/java/io/camunda/zeebe/test/util/bpmn/random/blocks/CallActivityBlockBuilder.java

```diff
@@ -64,7 +64,7 @@ public ExecutionPathSegment generateRandomExecutionPath(final ExecutionPathConte
     final ExecutionPathSegment result = new ExecutionPathSegment();
 
     if (calledProcessBuilder != null) {
-      result.append(calledProcessBuilder.findRandomExecutionPath(context));
+      result.append(calledProcessBuilder.findRandomExecutionPath(context), true);
```

Comment:
I will extract the variable and make it more clear this way. I feel the documentation is not necessary because of the javadoc I've added to the append method.